WORKSPACE

```diff
@@ -69,7 +69,14 @@ npm_repositories()
 npm_import(
     name = "acorn__8.4.0",
     integrity = "sha512-ULr0LDaEqQrMFGyQ3bhJkLsbtrQ8QibAseGZeaSUiT/6zb9IvIkomWHJIvgvwad+hinRAgsI51JcWk2yvwyL+w==",
+    # Packages where to link this npm dependency as a direct dependency
+    link_paths = [
```

Comment:
but it's not sufficient, since those packages also need a link_* call in their BUILD file? should there be a comment here that this allows the link but doesn't create it?

Reply:
Remote this altogether from the example; it is simpler allow npm_imports with the default of `link_paths = []` to be linked anywhere depending on the `direct` attribute of `link_js_package`. This makes the behaviour of linking a manual npm_import match the behaviour of linking a first party `js_package`.